assets/css/style.pcss

```diff
@@ -15,33 +15,47 @@
 	display: flex;
 	align-items: center;
 	justify-content: center;
+	width: var(--tribe-play-button-size);
+	height: var(--tribe-play-button-size);
 	position: absolute;
 	top: calc(50% - 0.5 * var(--tribe-play-button-size));
 	left: calc(50% - 0.5 * var(--tribe-play-button-size));
```

Comment:
❓ Could we use a more modern approach to centering? Maybe flexbox or grid that won't require the complex calcs and recentering at different breakpoints?

Reply:
@dpellenwood Updated to use grid - let me know if this is what you were looking for.

Reply:
Wonderful! No more absolutes and manual positioning!